### PR TITLE
[VEN-2554]: VIP for Prime Adjustment

### PR DESCRIPTION
**Pull request by Debugger022**
## Description

Resolves #VEN-2554
This PR contains VIP to update the distribution percentage for **USDTPrimeConverter** and **ETHPrimeConverter** through **PSR**.

### CODE REVIEW

File: vips/vip-289/bscmainnet.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+export const BTC_DISTRIBUTION_SPEED = parseUnits("27.7777777777", 10);
+
+export const ETH = "0x2170Ed0880ac9A755fd29B2688956BD959F933F8";
+export const ETH_DISTRIBUTION_SPEED = parseUnits("2752.7006172839", 10);
+
+export const USDC = "0x8AC76a51cc950d9822D68b83fE1Ad97B32Cd580d";
+export const USDC_DISTRIBUTION_SPEED = parseUnits("11574074.074074073", 10);
+
+export const USDT = "0x55d398326f99059fF775485246999027B3197955";
+export const USDT_DISTRIBUTION_SPEED = parseUnits("15432098.765432095", 10);
```
Comment (chechu) [inline vips/vip-289/bscmainnet.ts:22]:
```suggestion
export const USDT_DISTRIBUTION_SPEED = parseUnits("15432098.7654320987", 10);
```

File: vips/vip-289/bscmainnet.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+export const PLP = "0x23c4F844ffDdC6161174eB32c770D4D8C07833F2";
+
+export const BTC = "0x7130d2A12B9BCbFAe4f2634d864A1Ee1Ce3Ead9c";
+export const BTC_DISTRIBUTION_SPEED = parseUnits("27.7777777777", 10);
+
+export const ETH = "0x2170Ed0880ac9A755fd29B2688956BD959F933F8";
+export const ETH_DISTRIBUTION_SPEED = parseUnits("2752.7006172839", 10);
+
+export const USDC = "0x8AC76a51cc950d9822D68b83fE1Ad97B32Cd580d";
+export const USDC_DISTRIBUTION_SPEED = parseUnits("11574074.074074073", 10);
```
Comment (chechu) [inline vips/vip-289/bscmainnet.ts:19]:
```suggestion
export const USDC_DISTRIBUTION_SPEED = parseUnits("11574074.074074074", 10);
```

File: simulations/vip-289/bscmainnet.ts [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  testVip("VIP-289 Prime Adjustment", vip289(), {
+    callbackAfterExecution: async txResponse => {
+      await expectEvents(txResponse, [PSR_ABI], ["DistributionConfigUpdated"], [2]);
+      await expectEvents(txResponse, [PLP_ABI], ["TokenDistributionSpeedUpdated"], [4]);
+    },
+  });
+
+  describe("Post-Execution", () => {
+    it("should update the distribution config", async () => {
+      const distributionsLength = await psr.totalDistributions();
```
Comment (chechu) [inline simulations/vip-289/bscmainnet.ts:48]: I would confirm the number of distribution rules is the same before and after the VIP execution

File: vips/vip-289/bscmainnet.ts (hunk 1 above)
Comment (Debugger022) [reply vips/vip-289/bscmainnet.ts]: Done https://github.com/VenusProtocol/vips/pull/261/commits/b6c17c3c0a8b83de463d8f53644b04841ce18a23

File: simulations/vip-289/bscmainnet.ts (hunk 3 above)
Comment (Debugger022) [reply simulations/vip-289/bscmainnet.ts:48]: Done https://github.com/VenusProtocol/vips/pull/261/commits/c80b6817a03e66fdc08f37c638547d9416de3140

**Review by chechu — COMMENTED** [review]
lgtm

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/291?chainId=56

**Pull request merged**